song.h

```diff
@@ -0,0 +1,25 @@
+#ifndef SONG_H
+#define SONG_H
+
+#include <iostream>
+
+using namespace std;
+
+class Song
+{
+  friend ostream& operator<<(ostream& os, const Song& song);
+  friend istream& operator>>(istream& is, Song& song);
+  friend bool operator==(const Song& lhs, const Song& rhs);
+
+public:
+    Song();
+    Song(const char* t, const char* a);

```

Comment:
don't use abbreviations. 
```suggestion
    Song(const char* title, const char* artist);

```

Reply:
I didn't write the "Song" file.